docs/networks/software-versions.mdx

```diff
@@ -12,6 +12,49 @@ Release candidates are software releases that are also released to the [Testnet]
 
 [testnet]: ./README.mdx
 
+## Protocol 23 (Testnet only, Jul 17, 2025)
+
+### Software
+
+| Software                   | Version                             |
+| -------------------------- | ----------------------------------- |
+| XDR                        | `todo`                              |
```

Comment:
XDR: `v23.0`
Rust XDR: `23.0.0-rc.2`
Soroban Environment: `23.0.0-rc.2`
Stellar Core: `23.0.0rc4` (note: we're using the tags in this doc; Horizon needs to be updated as well)
Soroban Rust SDK: `23.0.0-rc.2.2`

Reply:
yes, the sw tags are the pattern, but it seems like the deb pkgs are really important as well, we should probably include both here?

Reply:
ok, I'm adding a 3rd column here for 'Package' so can accommodate both versions.